include/overlay021/struct_ov21_021E6A68.h

```diff
@@ -3,7 +3,7 @@
 
 typedef struct {
     int unk_00;
-    int unk_04;
+    int heapID;
```

Comment:
**polish:** Use `enum HeapId` here from `include/constants/heap.h` if possible.

Reply:
Do you mean to change "int heapID" to "HeapID heapID"? The enum also appears to be incomplete; would that interfere with unidentified IDs?

Reply:
1. Yes, `enum HeapID heapID;`
2. Nope; enums are set up to compile to `int` in this repo.